main.tex

```diff
@@ -343,28 +343,40 @@ \subsubsection{Normalization layers}
 
 Batch, instance, and layer normalization are often informally described using the same
 equation, but they each correspond to very different functions. They differ
-by which axes are normalized.
+both by which axes are \textit{standardized} as well as their parameters.
 
-We can define a single generic normalization layer:
+We can define a single generic standardization function as:
 \begin{align*}
-  \nfun{\ax}{XNorm} \colon \mathbb{R}^{\ax} &\rightarrow \mathbb{R}^{\ax} \\
-  \nfun{\ax}{XNorm}(X; \gamma, \beta, \epsilon) &= \frac{X - \nfun{\ax}{mean}(X)}{\sqrt{\nfun{\ax}{var}(X)} + \epsilon} \odot \gamma + \beta
+  \nfun{\ax}{standardize} \colon \mathbb{R}^{\ax} &\rightarrow \mathbb{R}^{\ax} \\
+  \nfun{\ax}{standardize}(X) &= \frac{X - \nfun{\ax}{mean}(X)}{\sqrt{\nfun{\ax}{var}(X) + \epsilon}}
 \end{align*}
 where
 \begin{align*}
-  \gamma, \beta &\in \mathbb{R}^{\ax} \\
   \epsilon &> 0.
 \end{align*}
 
-Now, suppose that the input has three axes:
+Now, consider norm functions each with the same type.
 \begin{align*}
-X &\in \reals^{{\batch \times \chans \times \layer}}
+\text{xNorm} &: \reals^{{\batch \times \chans \times \layer}} \rightarrow \reals^{{\batch \times \chans \times \layer}}
 \end{align*}
 Then the three kinds of normalization layers can be written as:
 \begin{align*}
-Y &= \nfun{\batch}{XNorm}(X; \gamma, \beta) && \text{batch normalization} \\
-Y &= \nfun{\layer}{XNorm}(X; \gamma, \beta) && \text{instance normalization} \\
-Y &= \nfun{\layer,\chans}{XNorm}(X; \gamma, \beta) && \text{layer normalization}
+\text{BatchNorm}(X; \gamma, \beta) &= \nfun{\batch,\layer}{standardize}(X) \ndot{} \gamma + \beta \text{ where }   \gamma, \beta \in \mathbb{R}^{\chans} \\
```

Comment:
can we insert some &'s to align the side conditions?

Reply:
fixed